code/fundamentals.rb

```diff
@@ -7,9 +7,9 @@
 puts message
 
 # 1. Create a variable `my_favorite_animal` and assign it the name of your favorite animal
-
+my_favorite_animal="horse"
 # 2. Print "My favorite animal is <your-favorite-animal>" by printing a string with `my_favorite_animal` interpolated into it.
-
+"My favorite animal is #{my_favorite_animal}"
```

Comment:
You formed the string correctly 👍 , but to print it out we need to use the `puts` method:

`puts "My favorite animal is #{my_favorite_animal}"`